release-calendar/src/client/components/App.tsx

```diff
@@ -13,10 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import * as React from 'react';
+import {Calendar} from './Calendar';
+//TODO: remove DATA and instead populate with data from ./test/development-data.ts
+import {DATA} from '../models (TODO: rename folder)/data';
+import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
 import {Header} from './Header';
+import {getEvents} from '../models (TODO: rename folder)/fakeEvents';
+
+interface AppState {
+  events: EventSourceInput[];
+}
 
-export const App = (): JSX.Element => {
-  return <Header title='AMP Release Calendar' />;
-};
+export class App extends React.Component<{}, AppState> {
+  constructor(props: unknown) {
+    super(props);
+    this.state = {
+      events: [],
+    };
+  }
+  //TODO: change fetch API call to service
+  componentDidMount(): void {
+    fetch('https://jsonplaceholder.typicode.com/todos')
```

Comment:
Instead of doing a real fetch here, you can do a `Promise.resolve(getEvents(DATA)).then()...`